src/reducers/selectors.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+
+import { createSelector } from 'reselect';
+
+export const EMPTY_OBJ = {};
+export const EMPTY_AR = [];
+
+// Checks a response object for errors and returns them
+export const processErrors = (responseValue: ?QueryResponse): Object => {
+	if (!responseValue || !responseValue.error) {
```

Comment:
This was always a confusing method to me as well. It's used in 11 places in mup-web (some comments, conversations, photos code), so I'm not confident in removing/changing its behavior in a way that will guarantee to work with those implementations - definitely something that could/should be cleaned up in a separate ticket, though.